### ISSUE TITLE
About the high cpu occupation

### ISSUE THREAD

**Issue by czecze123** [Xflick/EEND_PyTorch#9]
Hi, Sorry to disturb you, first of all thanks for the implementation of the EEND system in pytorch. Recently, when i training the large model ,i found that the CPU occupation is really high. i want to know the reason of it and how to solve it.
![image](https://user-images.githubusercontent.com/71816461/139264311-d77eebf8-eb76-4a99-91f8-dfa06e9c58be.png)

**Comment by czecze123** [comment]
hi, sorry to disturb you again. when i was training the model using my own data, i found that the training loss can't go down at 0.16, i want to ask you two questions: (1) about training dataset audio number,  i used 20000 audios made by the code. what is your training dataset number? (2) about your training loss, i want to ask what is your final training loss and dev loss. Thanks!
